src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -84,6 +85,7 @@
 #include <EnergyPlus/OutputProcessor.hh>
 #include <EnergyPlus/OutputReportPredefined.hh>
 #include <EnergyPlus/ScheduleManager.hh>
+#include <EnergyPlus/SolarShading.hh>
 #include <EnergyPlus/SurfaceGeometry.hh>
```

Comment:
This is added to access `GetShadowingInput()` defined in the Solar Shading module.